go/chat/push.go

```diff
@@ -501,6 +501,13 @@ func (g *PushHandler) Activity(ctx context.Context, m gregor.OutOfBandMessage) (
 				g.Debug(ctx, "chat activity: newMessage: outboxID is empty")
 			}
 
+			// Coin flip manager can completely handle the incoming message
+			if g.G().CoinFlipManager.MaybeInjectFlipMessage(ctx, nm.Message, nm.InboxVers, uid, nm.ConvID,
+				nm.TopicType) {
+				g.Debug(ctx, "chat activity: flip message handled, early out")
+				return
```

Comment:
why not allow the rest of the processing so we don't have to worry about weird state with these functions not being called. could return a `*chat1.MessageUnboxed` and have a `ConvSource.PushUnboxed`?

Reply:
(or just unbox twice if we're not worried about the perf there)

Reply:
The point is to skip this processing, I don't think it is necessary for these. The only thing we absolutely need to do is update the inbox version, which this guy does.